### PR TITLE
Migration to 3.1 net core

### PR DESCRIPTION
**Pull request by bdelicexecom**
Migration from .NET Core 2.1.403 to .NET Core 3.1.201.

Due to major update, following staff are changed:

- Instead of Json.NET, ASP.NET Core 3.1 and .NET Core 3.1 include a different JSON API that focuses a bit more on performance. In order to reconfigure our ASP.NET Core 3.1 project with Json.NET, we added a NuGet reference to Microsoft.AspNetCore.Mvc.NewtonsoftJson, which is the package that includes all the necessary bits. Then, in the Startup’s ConfigureServices we added "services.AddNewtonsoftJson();"

- Instead Microsoft.AspNetCore.All, .NET Core use now Microsoft.AspNetCore.App.

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
   "sdk": {
-    "version": "2.1.403"
+    "version": 3.1.201"
```
Comment (stanislavgrujic) [inline global.json:3]: Is this JSON valid? It seems to me that you are either missing an opening quotation mark before the value, or have a quotation mark extra at the end of the value.

File: Exebite.API/Logs/internal-nlog.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+2020-05-04 03:54:56.2722 Info Adding target File Target[allfile]
+2020-05-04 03:54:56.3514 Info Adding target File Target[ownFile-web]
+2020-05-04 03:54:56.3823 Info Configured from an XML element in C:\Users\Bojan Delic\source\repos\execom-eu\exebite\Exebite.API\bin\Debug\netcoreapp2.1\nlog.config...
+2020-05-04 03:54:56.4261 Info Found 45 configuration items
+2020-05-04 10:48:30.3620 Info Message Template Auto Format enabled
+2020-05-04 10:48:30.4672 Info Loading assembly: NLog.Web.AspNetCore
+2020-05-04 10:48:31.2083 Info Adding target File Target[allfile]
+2020-05-04 10:48:32.0355 Info Adding target File Target[ownFile-web]
+2020-05-04 10:48:32.2936 Info Configured from an XML element in C:\Users\Bojan Delic\Source\Repos\execom-eu\exebite\Exebite.API\bin\Debug\netcoreapp2.1\nlog.config...
+2020-05-04 10:48:32.6183 Info Found 45 configuration items
```
Comment (stanislavgrujic) [inline Exebite.API/Logs/internal-nlog.txt:12]: This looks as a log file. Is it really needed to have it in the source control. I would expect that no log files are committed.

File: global.json (hunk 1 above)
Comment (bdelicexecom) [reply global.json]: Resolved. Yes, missing opening quotation.

File: Exebite.API/Logs/internal-nlog.txt (hunk 2 above)
Comment (bdelicexecom) [reply Exebite.API/Logs/internal-nlog.txt]: Removed.

**Pull request merged**